code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -1607,7 +1610,12 @@ void control_config_common_read_section(int s, bool first_override) {
 				key = -1;
 			} else {
 				stuff_string(szTempBuffer, F_NAME);
-				key = mKeyNameToVal[szTempBuffer];
+				try{
+					key = mKeyNameToVal.at(szTempBuffer);
+				}
+				catch(std::out_of_range&){
+					Warning(LOCATION,"Table tried to set default for a key that doesn't exist: %s",szTempBuffer.c_str());
```

Comment:
Please reference the .tbl option in the warning message.  This should give mod devs a better clue of whats going wrong.

ex:
"Table tried to set \'$Key Default\' for a key that doesn't exist: %s"

(If you can come up with a better looking style then by all means use it :smile:)